smoke-tests-otel-starter/spring-smoke-testing/src/main/java/io/opentelemetry/spring/smoketest/SpringSmokeOtelConfiguration.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.spring.smoketest;
+
+import io.opentelemetry.instrumentation.spring.autoconfigure.OpenTelemetryInjector;
+import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizerProvider;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.logs.ConfigurableLogRecordExporterProvider;
+import io.opentelemetry.sdk.autoconfigure.spi.metrics.ConfigurableMetricExporterProvider;
+import io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.util.Map;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+public class SpringSmokeOtelConfiguration {
+
+  @Bean
+  OpenTelemetryInjector openTelemetryInjector() {
+    return SpringSmokeInstrumentationExtension::init;
+  }
+
+  @Bean
+  AutoConfigurationCustomizerProvider autoConfigurationCustomizerProvider() {
+    return provider -> provider.addPropertiesSupplier(SpringSmokeOtelConfiguration::getProperties);
+  }
+
+  private static Map<String, String> getProperties() {
+    return Map.of(
+        // We set the export interval of the metrics to 100 ms. The default value is 1 minute.
+        "otel.metric.export.interval", "100",
+        // We set the export interval of the spans to 100 ms. The default value is 5 seconds.
+        "otel.bsp.schedule.delay", "100",
+        // We set the export interval of the logs to 100 ms. The default value is 1 second.
+        "otel.blrp.schedule.delay", "100",
+        "otel.traces.exporter", "memory",
```

Comment:
This property and the two following ones should ideally be in the same project as the `ConfigurableLog/Trace/MetricExporterProvider` because of the `memory` string.

Reply:
it's even in the same class - 6 occurrences of "memory"

Reply:
Oh, sorry

Reply:
I'll extract a constant